android/expoview/src/main/java/host/exp/exponent/kernel/services/ErrorRecoveryManager.java

```diff
@@ -11,38 +11,38 @@
 import javax.inject.Inject;
 
 import host.exp.exponent.di.NativeModuleDepsProvider;
-import host.exp.exponent.kernel.ExperienceId;
+import host.exp.exponent.kernel.ExperienceKey;
 import host.exp.exponent.storage.ExponentSharedPreferences;
 
 public class ErrorRecoveryManager {
 
   private static final long FIVE_MINUTES_MS = 5 * 60 * 1000;
   private static final long AUTO_RELOAD_BUFFER_BASE_MS = 5 * 1000;
 
-  private static Map<ExperienceId, ErrorRecoveryManager> sExperienceIdToManager = new HashMap<>();
+  private static Map<ExperienceKey, ErrorRecoveryManager> sExperienceKeyToManager = new HashMap<>();
```

Comment:
I think this should be a map from scope key to ErrorRecoveryManager because conceptually speaking, if someone were to rename/transfer a project, the scope key would remain constant and be the value we want to use as the key. In other words, we want to scope ErrorRecoveryManagers so scope key is best.